src/applications/post-911-gib-status/tests/containers/Post911GIBStatusApp.unit.spec.jsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 import { expect } from 'chai';
+import { Provider } from 'react-redux';
+import createCommonStore from '@department-of-veterans-affairs/platform-startup/store';
 
 import Post911GIBStatusApp from '../../containers/Post911GIBStatusApp';
-import reducer from '../../reducers/index.js';
-import createCommonStore from 'platform/startup/store';
+import reducer from '../../reducers/index';
 
 const store = createCommonStore(reducer);
 
 describe('<Post911GIBStatusApp>', () => {
   it('should render', () => {
-    const tree = shallow(<Post911GIBStatusApp store={store} />);
+    const tree = mount(
+      <Provider store={store}>
+        <Post911GIBStatusApp />
+      </Provider>,
+    );
```

Comment:
Have to actually mount this component now, because the `useFeatureToggle` functionality requires it.